vllm/model_executor/models/chatglm.py

```diff
@@ -0,0 +1,329 @@
+from typing import List, Optional, Tuple
```

Comment:
add a copy right notice here？
and `Inference-only ChatGLM model compatible...` be consistent with other models.